Makefile

```diff
@@ -70,16 +68,15 @@ misc/certs/ca-certificates.crt:
 	docker build -t "amazon/amazon-ecs-agent-cert-source:make" misc/certs/
 	docker run "amazon/amazon-ecs-agent-cert-source:make" cat /etc/ssl/certs/ca-certificates.crt > misc/certs/ca-certificates.crt
 
-short-test: gogenerate
-	cd agent && godep go test -short -timeout=25s -v -cover ./...
-	@git checkout -- agent/version/version.go
+short-test:
+	. ./scripts/shared_env && go test -short -timeout=25s -v ./agent/...
```

Comment:
I can do it in the bash scripts. Makefile's commands are interpreted with `sh` which does not have the `source` builtin. I don't know of a more readable alternative to `.` in `sh`